org.natron.Natron.json

```diff
@@ -0,0 +1,841 @@
+                "make install"
+            ],
+            "cleanup": [
+                "/include"
+            ]
+        },
+        {
+            "name": "seexpr",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=release",
+                "-DCMAKE_INSTALL_LIBDIR=lib"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/wdas/SeExpr/archive/v1-2.11.tar.gz",
+                    "sha256": "b1395e1e87bdc75b6abc65ab8c4894b1e04ddb60c6eef277607bb6d44b31a6c2"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/seexpr-nodemos-notests.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/seexpr-fix-inline-error.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/seexpr-fix-cxxflags.patch"
+                }
+            ],
+            "cleanup": [
+                "src/SeExprEditor/generated/",
+                "src/SeExpr/generated/"
+            ]
+        },
+        {
+            "name": "natron",
+            "buildsystem": "qmake",
+            "no-autogen": true,
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/NatronGitHub/Natron.git",
+                    "tag": "2.3.10",
+                    "commit": "c80671898b39051516df57e7f8ec23c10e856ed5"
+                },
+                {
+                    "type": "file",
+                    "path": "config.pri"
+                }
+            ],
+            "build-commands": [
+                "qmake -r",
+                "make"
+            ]
+        },
+        {
+            "name": "openfx-misc",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/devernay/openfx-misc.git",
+                    "tag": "Natron-2.3.10",
+                    "commit": "3ab053146dafcbfac5b687f311d75bd59db511d8"
+                }
+            ],
+            "build-commands": [
+                "make CONFIG=release"
+            ],
+            "cleanup": [
+                "openfx/Documentation",
+                "openfx/Examples",
+                "openfx/website",
+                "openfx/scripts",
+                "openfx/Guide",
+                "openfx/HostSupport/expat-2.2.1",
+                "openfx/HostSupport/examples",
+                "openfx/Support/PropTester"
+            ]
+        },
+        {
+            "name": "openfx-io",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/NatronGitHub/openfx-io.git",
```

Comment:
please also add openfx-arena and openfx-gmic for the final version

Reply:
Ok